src/app/store/recipes/effects/load-success-recipe.effect.ts

```diff
@@ -0,0 +1,31 @@
+import { Injectable } from '@angular/core';
+import { AngularFirestore } from '@angular/fire/compat/firestore';
+import Recipe from '@models/recipe.model';
+import { Actions, createEffect, ofType } from '@ngrx/effects';
+import { map, mergeMap, of, switchMap, tap } from 'rxjs';
+import RecipesActions from '../recipes.actions';
+
+@Injectable()
+export class loadRecipesSuccessEffect {
+	loadRecipes$ = createEffect(() => {
+		return this.actions$.pipe(
+			ofType(RecipesActions.loadRecipesSuccess),
+			map(action => {
+				const { recipe } = action;
+				switch (action.action) {
+					case 'added':
```

Comment:
declare an Enum and use it here

Added = 'added' etc.

Reply:
I use DocumentChangeType so is type save and vscode autocompletion work